src/data/ACTIONS/RDM.js

```diff
@@ -258,7 +258,7 @@ export default {
 		onGcd: true,
 		cooldown: 2.5,
 		castTime: 2,
-		potency: 270, //Increase Black by 9
+		potency: 350, //Increase Black by 9
```

Comment:
Dunno if it matters for analysis, but for correctness - per garland, it looks like you got Fire/Stone and Aero/Thunder flipped, potency-wise (Fire/Stone should be 300, Aero/Thunder should be 350).

Reply:
Indeed I did, I will correct it.